src/main/java/org/candlepin/subscriptions/tally/collector/DefaultProductUsageCollector.java

```diff
@@ -75,10 +96,14 @@ else if (!normalizedFacts.isVirtual()) {
   }
 
   @Override
-  public Optional<HostTallyBucket> collectForHypervisor(
-      String orgId, UsageCalculation prodCalc, NormalizedFacts hypervisorFacts) {
-
+  public Optional<HostTallyBucket> buildBucketForHypervisor(
+      UsageCalculation.Key key, NormalizedFacts hypervisorFacts) {
     /* do nothing for hypervisor-guest mappings by default */
     return Optional.empty();
   }
+
+  @Override
+  public void collectForHypervisor(UsageCalculation prodCalc, NormalizedFacts hypervisorFacts) {
+    throw new NotImplementedException("Not implemented for " + this.getClass().getName());
```

Comment:
I think this will cause issues when collecting for Satellite, or other non-RHEL products using data from HBI? I'd rather see this intentionally empty.